references/detection/transforms.py

```diff
@@ -437,3 +439,99 @@ def forward(
                 )
 
         return image, target
+
+
+def copy_paste(image, source_data, paste_image, paste_data, inplace=True):
+
+    image = image.clone() if not inplace else image
+    source_data = copy.deepcopy(source_data) if not inplace else source_data
+
+    number_of_masks = len(paste_data["masks"])
+    random_selection = torch.randint(0, number_of_masks, (number_of_masks,)).unique()
+
+    # Combine masks
+    paste_alpha_mask = paste_data["masks"][random_selection].sum(dim=0).greater(0)
+
+    paste_alpha_mask = F.gaussian_blur(paste_alpha_mask.unsqueeze(0), kernel_size=(5, 5), sigma=2.0)  # add blur
+
+    # Paste pixels from paste_image to image
+    image.mul_(torch.logical_not(paste_alpha_mask))  # Delete pixels from source image
+    paste_pixels = paste_image * paste_alpha_mask  # Copy pixels from paste image
+    image.add_(paste_pixels)  # paste it on source image
+
+    # update original masks
+    source_data["masks"] = source_data["masks"] ^ paste_alpha_mask & source_data["masks"]
```

Comment:
Any objections why this line to update original masks can't be a single op like:
```python
source_data["masks"] = source_data["masks"] * (~paste_alpha_mask)
```
?